packages/frontend-api/src/Resources/config/graphql-types/OrderItemDecorator.types.yml

```diff
@@ -0,0 +1,24 @@
+OrderItemDecorator:
+    type: object
+    decorator: true
+    config:
+        description: "Represent one item in the order"
+        fields:
```

Comment:
Hmm... Interesting idea. Can you please explain to me the need for it? 
I don't think we want to query some order items by UUID, or it can be somehow used in the storefront?

Reply:
I'm thinking about the options to confirm delivery of items, but it's just an idea ...